src/istio/zarf.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ZarfPackageConfig
```

Comment:
Let me know what you think about these changes - https://github.com/defenseunicorns/uds-core/pull/108 - I think this feels closer to the common pattern with other src packages?

Reply:
merged. I agree with that pattern i had started going down that road and then got to the gateways which didnt fit and figure i'd try to get feedback first.